smdebug/core/config_constants.py

```diff
@@ -1,5 +1,5 @@
 CONFIG_FILE_PATH_ENV_STR = "SMDEBUG_CONFIG_FILE_PATH"
-CONFIG_DEFAULT_WORKER_NAME = "worker_0"
+DEFAULT_WORKER_NAME = "worker_0"
```

Comment:
Rename to `DEFAULT_CHIEF_WORKER_NAME` ?

Reply:
Is it only as chief worker? Chief only makes sense in distributed right?

Reply:
Hmm, you're right, resolving this